enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -127,6 +129,34 @@ func clonePrivateTestRepo(t *testing.T) string {
 	return repoDir
 }
 
+func newUUID(t *testing.T) string {
+	id, err := uuid.NewRandom()
+	require.NoError(t, err)
+	return id.String()
+}
```

Comment:
I think we also have our own `uuid` package under server/util that never returns an error if you want to use that instead